src/View/layout.html.twig

```diff
@@ -8,12 +10,20 @@
 		<link rel="stylesheet" href="/assets/css/header.css">
 
 
-		<title>
-			{% block title %}GooDealMap
-			{% endblock %}
-		</title>
 
-		{% block stylesheet %}{% endblock %}
+    <title>{% block title %}GoodealMap{% endblock %}</title>
+		
+		{% block stylesheet %}
+        <link rel="stylesheet" href="/assets/css/reset.css">
+        <link rel="stylesheet" href="/assets/css/style.css">
+        <link rel="stylesheet" href="/assets/css/mapDesktop.css">
```

Comment:
this file and some others should be injected only in index file, please sort those files and separate'em from the layout